Tests/SentryTests/Transaction/SentrySpanTests.swift

```diff
@@ -153,6 +159,18 @@ class SentrySpanTests: XCTestCase {
         XCTAssertEqual(childSpan.context.operation, fixture.someOperation)
         XCTAssertEqual(childSpan.context.spanDescription, fixture.someDescription)
     }
+
+    func testStartChildOnFinishedSpan() {
```

Comment:
`m`: I think we should also add a test that this logic works for a grandchild. So basically, it should work as well when
```Swift
let span = fixture.getSut()
let childSpan = span.startChild(operation: fixture.someOperation)
childSpan.finish()
span.finish()
let grandChild = childSpan.startChild(operation: fixture.someOperation, description: fixture.someDescription)
```